content/_code-samples/delete-account/js/delete-account.js

```diff
@@ -0,0 +1,65 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Blackhole an account.
+  // For some addresses, it's possible that no one has the secret key, in which case the account is a black hole and the XRP is lost forever.
+  // https://xrpl.org/accounts.html#special-addresses
+  // https://xrpl.org/disable-master-key-pair.html#disable-master-key-pair
+
+  async function main() {
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
+  
+    // Get account credentials from the Testnet Faucet
+    console.log("Requesting addresses from the Testnet faucet...")
+    const { wallet, balance } = await client.fundWallet()
+    
+    console.log("\nAccount: ", wallet.address)
+    console.log("   Seed: ", wallet.seed)
+
+    // Send AccountSet transaction 
+    const SetRegularKey_tx = await client.autofill({
+      "TransactionType": "SetRegularKey",
+      "Account": wallet.address,
+      "RegularKey": "rrrrrrrrrrrrrrrrrrrrBZbvji" 
+    })
+  
+    const SetRegularKey_tx_signed = wallet.sign(SetRegularKey_tx)  
+    const SetRegularKey_tx_result = await client.submitAndWait(SetRegularKey_tx_signed.tx_blob)
+
+    console.log("\n            Transaction hash:", SetRegularKey_tx_signed.hash)
+    console.log(" SetRegularKey Submit result:", SetRegularKey_tx_result.result.meta.TransactionResult)
```

Comment:
Also recommend rewording the SetRegularKey line to be more explicit about what action the code is taking. 
```suggestion
console.log(`\n Submitted a SetRegularKey transaction (Result: ${SetRegularKey_tx_result.result.meta.TransactionResult})`)
console.log("Transaction hash:", SetRegularKey_tx_signed.hash)

```